webaudio-offlinecontext-rendering/index.html

```diff
@@ -0,0 +1,133 @@
+
+    <link rel="icon" href="../images/favicon.ico">
+
+    <link rel="stylesheet" href="../styles/main.css">
+  </head>
+
+  <body>
+    <h1>WebAudio Offline context rendering promise Sample</h1>
+
+    <p>Available in <a href="http://www.chromestatus.com/feature/5692188675538944">Chrome 42+</a></p>
+
+    <p>
+      The <code>OfflineAudioContext.startRendering</code> method renders audio to an <code>AudioBuffer</code> potentially faster than real-time.
+      This method returns a <code>Promise</code> which is fulfilled when the rendering is complete.
+      It is further documented in the <a href="http://webaudio.github.io/web-audio-api/#widl-OfflineAudioContext-startRendering-Promise-AudioBuffer">Web Audio API</a>.
+    </p>
+
+    <div class="output">
+      <div>
+        Merge two tones, display information about the final <code>AudioBuffer</code>, and plays it.
+      </div>
+      <button id="start">Start</button>
+      <pre id="results"></pre>
+    </div>
+
+    <!-- // [START code-block] -->
+    <script>
+
+    var seconds = 4;
+    var offlineAudioContext = new OfflineAudioContext(2, seconds * 44100, 44100);
+    var audioContext = new AudioContext();
+
+    // Fade the tones in/out.
+    var gainNode = offlineAudioContext.createGain();
+    gainNode.connect(offlineAudioContext.destination);
+    gainNode.gain.linearRampToValueAtTime(0, 0);
+    gainNode.gain.linearRampToValueAtTime(1, 0.5);
+    gainNode.gain.linearRampToValueAtTime(1, seconds - 1.0);
+    gainNode.gain.linearRampToValueAtTime(0, seconds - 0.5);
+
+    var firstSource = offlineAudioContext.createOscillator();
+    firstSource.frequency.value = 200;
+    firstSource.connect(gainNode);
+    firstSource.start();
+
+    var secondSource = offlineAudioContext.createOscillator();
+    secondSource.frequency.value = 350;
+    secondSource.connect(gainNode);
+    secondSource.start();
+
+    var results = document.getElementById('results');
+    var button = document.getElementById('start');
+
+    button.addEventListener('click', function() {
+      button.remove();  // remove button from DOM
+
+      var start = +new Date();
+
+      // Render the context, returning a Promise.
+      var p = offlineAudioContext.startRendering();
+      p.then(function(audioBuffer) {
+        console.info('audio result is', audioBuffer);
+
+        var duration = +new Date() - start;
+        results.innerText = audioBuffer.toString() + ' in ' + duration + 'ms';
+
+        // Transform to an AudioBufferSourceNode.
+        var source = audioContext.createBufferSource();
+        source.buffer = audioBuffer;
+        source.connect(audioContext.destination);
+        return source;
+      }).then(function(source) {
+        // Play the source.
+        source.start();
+      });
+
+    });
+
+    </script>
+    <!-- // [END code-block] -->
+
+    <script>
+      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```

Comment:
What is the target (or the owner of account) of this analytic code? I think this should be removed, because we'll inject the Web Audio API team's analytic code here.